### DISCUSSION TITLE
dude...

### DISCUSSION THREAD

**Discussion by jbxskm** [BigfootsGS/BigfootsGS.github.io#45] [General]
if you just made those links into github.io links you could've saved alotta cash

**Comment by jbxskm** [comment]
i think
i got introduced to github recently. i dont even know how to make a github.io website.

**Reply by Bigfoot9999** [reply]
> if you just made those links into github.io links you could've saved alotta cash

- All of our hosting is 100% free thanks to CloudFlare and Freenom
- To create a .github.io link, go to a repo's settings, then click pages, then configure Github Pages. 
- If you need any help join my Discord Server: https://bgs.pages.dev/invite, and we can talk there.

**Comment by jbxskm** [comment]
cool
